api/v1alpha1/addresspool_types.go

```diff
@@ -42,10 +42,19 @@ type AddressPoolSpec struct {
 	AutoAssign bool `json:"autoAssign,omitempty"`
 }
 
+// AddressPool Services information
+type AddressPoolSvcStats struct {
+	// Service Name
+	Service string `json:"service,omitempty"`
+
+	// Service's Allocated Externat IP address
+	ExternalIP string `json:"external-ip,omitempty"`
+}
+
 // AddressPoolStatus defines the observed state of AddressPool
 type AddressPoolStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	AddressPoolServices []AddressPoolSvcStats `json:"address-pool-services"`
```

Comment:
this one should be I think `addressPoolServices`

Reply:
In general, maybe is better to have a map here?
If not for every event we will need to go over all the list to find if the service exist or need to be added

Reply:
Last point is how the operator will be updated about this? will it have a reconcile loop for services?
The metallb already have one, is that operator the one that will update this status or we are going to duplicate

Reply:
> this one should be I think `addressPoolServices`

JSON fields new to start with Cap letters

Reply:
CR controller need to update the status, I asked in the global slack to see if there is a way to retrieve this info from Metallb, it seems its not there, so I was thinking of add new informers to metallb-operator to monitor svc of type loadbalancer and the implement add/delete handlers that update the CR's status